### ISSUE TITLE
I want to add github stats section as well

### ISSUE THREAD

**Issue by theterrificm** [Ahsankhalid618/SkillSync#7]
Hi, I reviewed the application by setting up successfully and running it. The app looks nice, but I want to add github stats for the users as well: for e.g. how many stars they've received, how many number of commits they've pushed, etc.

How about that? Coulf you please assign me the feature request in order to begin the work?

Thanks in advance

**Comment by Ahsankhalid618** [comment]
@theterrificm Yeah sure go ahead, and post your changes screenshot here before raising a PR, Thanks.

**Comment by theterrificm** [comment]
That's great to know, Ahsan! I'll surely follow those instructions thanks

**Comment by Ahsankhalid618** [comment]
@theterrificm any progress?

**Comment by Tanmay-Chandgude** [comment]
hello @Ahsankhalid618 i would like to work on these!

**Comment by theterrificm** [comment]
> @theterrificm any progress?

Hi Ahsan, I hope you are doing great. Please expect an update by tomorrow or Wednesday hopefully.

**Comment by theterrificm** [comment]
Hello Ahsan, since you asked for the screenshot first, here's the screenshot from ft section:

![image](https://github.com/user-attachments/assets/7655e7dc-86fb-4de7-9404-07a07bbbae7c)


Let me know if can proceed to push?

Thanks in advance

**Comment by Ahsankhalid618** [comment]
@theterrificm Did you pick these Stats containers from Readme? if yes, These are, not aligned with the Project or its concept, It's just a single line of markdown to be added which is not needed in the Production project. GitHub Stats are a good thing to add in a project but not like that, Either create a react bases component (you can even choose any library like NextUI or ShadCN, etc)for stats and then render the stats in such a manner that it should align with the UI of the Project.
